notifications/src/main/java/com/dojo/notifications/service/notifierService/DockerNotifierService.java

```diff
@@ -81,9 +81,9 @@ public void executeRequests(Contest contest, String id, Object object, String me
         }
     }
 
-    private void notify(String receivers, String id, Contest contest, Object object, String message, NotificationType notificationType) {
+    private void notify(String receivers, String id, Contest contest, Object object, String message, NotificationType notificationType, int requestId) {
```

Comment:
if method args are getting too much and equivalent in a few places, we might need an object to move around instead of many parameters